README.md

```diff
@@ -97,6 +97,34 @@ class MyApp < Sinatra::Base
 end
 ```
 
+### If using Plain Ruby
+
+Rollbar isn't dependent on Rack or Rails for most of its functionality. In a regular script, assuming you've
+installed the rollbar gem:
+
+ 1. Require rollbar
+ 2. Configure rollbar
+ 3. Send Rollbar data
+
+```ruby
+require 'rollbar'
+
+Rollbar.configure do |config|
+  config.access_token = "SERVER_POST_ACCESS_TOKEN"
+  # Other Configuration Settings
+end
+
+begin
+  Rollbar.debug "Running Script"
+
+  run_script ARGV
+
+  Rollbar.info "Script ran successfully"
+rescue Exception => e
```

Comment:
Since rescuing `Exception` is a bad practice, let's remove it.

You can do just `rescue => e` here.